app/models/declaration.rb

```diff
@@ -0,0 +1,30 @@
+class Declaration < ApplicationRecord
+  belongs_to :application
+  belongs_to :cohort
+  belongs_to :lead_provider
+  belongs_to :superseded_by, class_name: "Declaration", optional: true
```

Comment:
can we also add the declaration to statement line items?

Reply:
do we need `participant_declaration_attempts` as well or are we dropping that intentionally?

Reply:
We are dropping them intentionally, as we don't need them yet